### PR TITLE
[218] Support  window function

### PR DESCRIPTION
**Pull request by lgbo-ustc**
Changelog category (leave one):
- New Feature

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...
Support window function

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by kyligence-git** [comment]
Can one of the admins verify this patch?

**Comment by lgbo-ustc** [comment]
test this pleasse with 720

**Comment by lgbo-ustc** [comment]
test this please with 720

File: utils/local-engine/Parser/SortRelParser.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#pragma once
```
Comment (zhanglistar) [inline utils/local-engine/Parser/SortRelParser.h:1]: 为啥多了个sort的？

Comment (lgbo-ustc) [reply utils/local-engine/Parser/SortRelParser.h:1]: 有段代码共用下。后续sort的实现会被这个替换掉

**Review by liuneng1994 — APPROVED** [review]
LGTM

**Pull request merged**